collect_app/src/main/java/org/odk/collect/android/formentry/FormEntryViewModel.java

```diff
@@ -83,14 +83,14 @@ public void addRepeat(boolean fromPrompt) {
         try {
             formController.newRepeat();
         } catch (RuntimeException e) {
-            error.setValue(e.getCause().getMessage());
+            error.setValue(e.getCause() != null ? e.getCause().getMessage() : "Unknown issue occurred while adding a new group");
```

Comment:
I think that if there's no cause, we should use the exception's own message. That should still provide more specific information.

Reply:
Ok done.